extension/services/deezer/deezer.js

```diff
@@ -7,6 +7,7 @@ Object.assign(Deezer, {
   id: "deezer",
   title: "Deezer",
   baseUrl: "https://www.deezer.com/",
+  imgSrc: "../../assets/images/deezer.svg",
```

Comment:
Since this could be used anywhere (e.g., in the popup), it's not clear what the relative path is relative to. Instead it should just be an absolute path (`/assets/images/deezer.svg`)